### PR TITLE
Added txpoolmaxsize

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added txpoolmaxsize

### CODE REVIEW

File: docs/Reference/Pantheon-CLI-Syntax.md [hunk 1]
```diff
@@ -952,6 +952,22 @@ The default is 8546. Ports must be [exposed appropriately](../Configuring-Panthe
...
+
+```bash tab="Example Command Line"
+--tx-pool-max-size=2000
+```
+
+```bash tab="Example Configuration File"
+tx-pool-max-size="2000"
+```
+
+Maximum number of transactions kept in the transaction pool. Default is 4096.
```
[the remark this answers is not in the document]
Comment (ajsutton) [reply docs/Reference/Pantheon-CLI-Syntax.md]: Not for long... https://github.com/PegaSysEng/pantheon/pull/1136 :)

File: docs/Using-Pantheon/Transactions/Transaction-Pool.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Transaction Pool 
+
+Options and methods for configuring and viewing the transaction pool include: 
+
+* [`txpool_pantheonTransactions`](../../Reference/JSON-RPC-API-Methods.md#txpool_pantheonTransactions) JSON-RPC API method
+
+* [`--tx-pool-max-size`](../../Reference/Pantheon-CLI-Syntax.md#tx-pool-max-size) command line option
+
+Once full, the Pantheon transaction pool accepts and retains local transactions in preference to remote transactions.
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Transaction-Pool.md]: Following up in prod dev but I'm pretty sure there's not a warning at this stage.

[the remark this answers is not in the document]
Comment (ajsutton) [reply docs/Using-Pantheon/Transactions/Transaction-Pool.md]: No there are no warnings.  Its *extremely* common to drop transactions from the pool.  There is a feature request for some way to subscribe to notifications of dropped transactions (probably a web sockets thing but eventually maybe even publishing to a Kafka queue).

File: docs/Using-Pantheon/Transactions/Transactions.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ All accounts and private keys in the examples are from the `dev.json` genesis fi
 
 ## eth_call or eth_sendRawTransaction
 
-You can interact with contracts using [eth_call](../Reference/JSON-RPC-API-Methods.md#eth_call) or [eth_sendRawTransaction](../Reference/JSON-RPC-API-Methods.md#eth_sendrawtransaction). 
+You can interact with contracts using [eth_call](../../Reference/JSON-RPC-API-Methods.md#eth_call) or [eth_sendRawTransaction](../../Reference/JSON-RPC-API-Methods.md#eth_sendrawtransaction). 
 
 |eth_call  | eth_sendRawTransaction |
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Using-Pantheon/Transactions/Transactions.md]: Done

**Pull request merged**